DEVOPS_PROJECT2/README.md

```diff
@@ -0,0 +1,30 @@
+Modules:
+
+The code is organized into different modules, each responsible for provisioning specific resources within the AWS infrastructure. These modules include EC2 instances, security groups, SSH key pairs, and a Virtual Private Cloud (VPC).
+EC2 Instances (main.tf):
```

Comment:
this is not way to write the readme file, please check on google and see how to write a README.md file for GIT